industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/egfr/PREPARATION_DETAILS.md

```diff
@@ -0,0 +1,8 @@
+1.  Protein structure was prepared following the maestro instructions: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/pull/89. Afterward NMA cappings were converted to NME.
+2.The structure did not contain any cofactors and no non-natural amino acids were found,
+therefore no modifications. 
+3. Ligands were reduced to the used ligands in: 10.1021/ja512751q;structures were left unchanged left unchanged, no conformational states were detected, but protonations changes 
```

Comment:
There's two instance of ligand 27 in the SDF file, and R and S enantiomer - is this expected?

Reply:
they were present here too, so I hope this is korrekt, right? (apparently makes no difference ) https://github.com/schrodinger/public_binding_free_energy_benchmark/blob/main/21_4_results/ligand_predictions/charge_annhil/egfr_pkacorr_out.csv

Reply:
kept only the R  variant -> should resolve this.